web/src/utils/factory-management/buildings.ts

```diff
@@ -36,27 +36,26 @@ export const calculateBuildingRequirements = (factory: Factory, gameData: DataIn
 }
 
 export const calculateBuildingsAndPower = (factory: Factory) => {
-  factory.totalPower = '0'
-  factory.buildingRequirements = {} as {[key: string]: BuildingRequirement }
+  factory.totalPower = 0
+  factory.buildingRequirements = {} as { [key: string]: BuildingRequirement }
 
   // Loop through each product and sum the power requirements based off the metrics already there.
   factory.products.forEach(product => {
     const building = product.buildingRequirements
     if (!factory.buildingRequirements[building.name]) {
       factory.buildingRequirements[building.name] = {
         name: building.name,
-        amount: '0',
+        amount: 0,
         powerPerBuilding: building.powerPerBuilding,
-        totalPower: '0',
+        totalPower: 0,
       }
     }
 
     const facBuilding = factory.buildingRequirements[building.name]
```

Comment:
@Maelstromeous - what does this constant do? I suspect it can be removed and is unused.

Reply:
For clarity, these lines can be removed I think:

```
    const facBuilding = factory.buildingRequirements[building.name]

    facBuilding.amount = facBuilding.amount + building.amount
    facBuilding.totalPower = facBuilding.totalPower + building.totalPower
```

Reply:
It's a reference to `factory.buildingRequirements[constructor], just a shortcut really for factory.buildingRequirements[consturctor].amount etc.

They're used for calculating the factory's building requirements, and I believe is very much needed.